app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
+    window.peopleList = new PeopleList
+    peopleList.fetch().then ->
+      $('.card').remove()
+      $.each(peopleList._byId, (_prop, person_data) ->
+        slug =  person_data.attributes._url.split('/')[2].split('.')[0]
```

Comment:
With backbone, try not to reference the `attributes` hash directly instead prefer using `get` and `set` so that the callback and change events trigger.   `model.get('_url')` for instance.